src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4607,7 +4607,7 @@ void GetDaylightingControls(EnergyPlusData &state, bool &ErrorsFound)
         daylightControl.Name = ipsc->cAlphaArgs(1);
 
         // Is it a space or zone name?
-        int const spaceNum = UtilityRoutines::FindItemInList(ipsc->cAlphaArgs(2), state.dataHeatBal->space);
+        int const spaceNum = Util::FindItemInList(state.dataIPShortCut->cAlphaArgs(2), state.dataHeatBal->space);
```

Comment:
Why not also delete the ipsc reference at line 4581 above?

Reply:
This branch has been ready since August but I didn't merge it because I didn't want to randomize the code-base while other people were getting their actual features and bug fixes in. As a result there were a lot more merge conflicts than usual, and this is probably the result of one of them.